web/test/pages/profile/profile-shared.test.tsx

```diff
@@ -386,6 +390,77 @@ describe("profile - shared", () => {
     });
   });
 
+  describe("Special Note Alert for Businesses Formed outside the Navigator", () => {
+    it.each(["FOREIGN", "STARTING"])(
```

Comment:
Since your using `.each` each test is run independently. There when the Foreign test is run, the `title - for FOREIGN and STARTING businesses` is not accurate. You can reference the the value of each array element in the title to resolve this. 

Link that was helpful for me - https://elfi-y.medium.com/reduce-boilerplate-test-code-with-jest-it-each-30a0eec9776d

Reply:
Thanks for the tip, thats something I totally forget to do more often than I should